src/main/java/com/gascalc/GasCalcController.java

```diff
@@ -10,8 +10,18 @@
 
 @Controller
 
+/**
+ * Main controller for GasCalc
+ * @author 
+ *
+ */
```

Comment:
Classes should have javadoc annotations. While this may seem unnecessary now, it could potentially save confusion once the project becomes more complex.

Best practices can be found at https://www.tutorialspoint.com/java/java_documentation.htm and https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html